src/aiod_sdk/endpoints/endpoint.py

```diff
@@ -0,0 +1,134 @@
+import abc
+import requests
+import urllib
+import pandas as pd
+from typing import Literal
+
+API_BASE_URL = "http://localhost/"
```

Comment:
We should make this easily configurable, from code and/or a configuration file or environment variable. Can just file this as an issue for later.

Reply:
Sure, I agree. I only kept it here thinking of implementing it in a config file during the next iteration/PR. Then, the next one would include an authentication logic before developing anything that requires user credentials, i.e. POST, PUT and DELETE calls. 
Does it make sense to you?